test/test_multidevice_pipeline.cpp

```diff
@@ -304,6 +304,122 @@ INSTANTIATE_TEST_SUITE_P(
         ::testing::Values(true),
         ::testing::Bool()));
 
+enum class SchedulingMode {
+  noIntraDeviceScheduling,
+  manualScheduling,
```

Comment:
```suggestion
  InterDeviceOnly,
  Manual,
```

For conciseness. "Scheduling" is unnecessary because the enum class will be used as `SchedulingMode::xyz`. I prefer `InterDeviceOnly` because it's positive and direct. (Of course, let me know if I misunderstand the meanings here).